main.go

```diff
@@ -234,6 +249,7 @@ type Config struct {
 	WatchedNamespaces    map[string]bool
 	ProbeAddr            string
 	GlobalAPISecret      client.ObjectKey
+	SyncPeriod           time.Duration
```

Comment:
not sure, why SyncPeriod passed from AtlasClusterReconciler, is this automatically will apply to all reconcilers

Reply:
Yes, it is applied to the manager so used for all reconcilers.